plugin/mutate/doc/design/test_mutant/pull_request.md

```diff
@@ -0,0 +1,73 @@
+# REQ-test_subset_of_mutants
+partof: REQ-test_mutant
+###
+
+The user wants to integrate mutation testing in a pull request workflow where
+time is crucial. The tool need to give feedback to the user within a short time
+frame such as 10-30 minutes.
+
+For this to work a number of tools need to be provided to the user.
+
+ * a report of only a diff which contains some of the alive mutants.
+ * The reported alive mutants should only be from the diff.
+ * the plugin should only test a subset of the unknown mutants. Which ones are
+   either specified manually by the user or read from a diff.
+
+# SPC-test_mutants_on_specified_lines
+partof: REQ-test_subset_of_mutants
```

Comment:
This seems very nice. Having the ability to choose which lines are mutated should really increase "user friendlyness".